### PR TITLE
Update community polyfills and tooling list

### PR DESCRIPTION
**Pull request by domenic**
See #146.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -546,18 +546,17 @@ The primary challenges here are in achieving agreement about built-in modules on
...
 * [@import-maps/resolve](https://www.npmjs.com/package/@import-maps/resolve) resolves a specifier relative to an import map.
 * [@jsenv/node-module-import-map](https://www.npmjs.com/package/@jsenv/node-module-import-map) generates an import map from your `package.json` and `node_modules/` directories.
-* [Snowpack](https://www.snowpack.dev) generates an import map when it installs your npm packages to run on the web.
-* [Built-in Module Demo (with Rollup)](https://glitch.com/edit/#!/rollup-built-in-modules) contains code for a [Rollup](https://rollupjs.org/) plugin that generates an import map, focused on built-in module polyfills.
+* [@jsenv/importmap-eslint-resolver](https://www.npmjs.com/package/@jsenv/importmap-eslint-resolver) enables import map resolution in [ESLint](https://eslint.org/)
 * [Deno](https://github.com/denoland/deno) is a JavaScript/TypeScript runtime with [built-in support for import maps](https://deno.land/manual/linking_to_external_code/import_maps).
 * [es-dev-server](https://www.npmjs.com/package/es-dev-server) allows using import maps during development, including polyfills.
```
Comment (daKmoR) [inline README.md:553]:
```suggestion
* [web-dev-server](https://www.npmjs.com/package/@web/dev-server) allows using import maps during development, including polyfills.
* [web-test-runner](https://www.npmjs.com/package/@web/test-runner) while testing you can mock external es modules via import maps
```

for reference some docs for Web Test Runner regarding mocking
https://modern-web.dev/docs/test-runner/writing-tests/mocking/

Comment (LarsDenBakker) [reply README.md:553]: To clarify: web-dev-server is the successor of es-dev-server :) We have a dedicated import map plugin for both the dev server and the test runner: https://modern-web.dev/docs/dev-server/plugins/import-maps/

File: README.md [hunk 2]
```diff
@@ -546,18 +546,17 @@ The primary challenges here are in achieving agreement about built-in modules on
 
 Several members of the community have been working on polyfills and tooling related to import maps. Here are the ones we know about:
 
-* [@import-maps/generate](https://www.npmjs.com/package/@import-maps/generate) generates an import map from your `yarn.lock`.
 * [@import-maps/resolve](https://www.npmjs.com/package/@import-maps/resolve) resolves a specifier relative to an import map.
 * [@jsenv/node-module-import-map](https://www.npmjs.com/package/@jsenv/node-module-import-map) generates an import map from your `package.json` and `node_modules/` directories.
-* [Snowpack](https://www.snowpack.dev) generates an import map when it installs your npm packages to run on the web.
-* [Built-in Module Demo (with Rollup)](https://glitch.com/edit/#!/rollup-built-in-modules) contains code for a [Rollup](https://rollupjs.org/) plugin that generates an import map, focused on built-in module polyfills.
+* [@jsenv/importmap-eslint-resolver](https://www.npmjs.com/package/@jsenv/importmap-eslint-resolver) enables import map resolution in [ESLint](https://eslint.org/)
```
Comment (dmail) [inline README.md:551]: Good for me.

For the record I have updated @jsenv/importmap-eslint-resolver readme to make it better, thanks for the link ;)

File: README.md (hunk 1 above)
Comment (domenic) [reply README.md:553]: Thanks! I'll consolidate these into a link to https://www.npmjs.com/package/@web/dev-server-import-maps

**Pull request merged**